samples/snippets/src/test/java/com/example/spanner/SequenceSampleIT.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner;
+
+import static com.example.spanner.SampleRunner.runSample;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import com.google.cloud.spanner.DatabaseId;
+import java.util.Collections;
+import java.util.concurrent.TimeUnit;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Integration tests for DML Returning samples for GoogleStandardSql dialect. */
+@RunWith(JUnit4.class)
+public class SequenceSampleIT extends SampleTestBase {
```

Comment:
If it's not too much effort, can we have a single integration test and parameterise it for PG/Google SQL dialects. If no, can you add what are the differences in the way both dialects are tested?

Reply:
Thanks for the suggestion. I've now parameterized the tests. Please take a look.